### PR TITLE
Fix GCE Windows Smoke Test and Support 2004 and 20H2.

### LINKED ISSUE #98285: Add Support for Windows Server 2004 and 20H2 for GCE Windows
**What would you like to be added**:
Add Support for Windows Server version 2004 and 20H2.

**Why is this needed**:
Windows Server version 1909 is going end of life.

### PR DESCRIPTION
**Pull request by jeremyje**
**What type of PR is this?**
/kind feature
/kind failing-test

**What this PR does / why we need it**:
Fix GCE Windows clusters `smoke-test.sh`.
Upgrade `pause-win` to run on newer version of Windows Server.

**Which issue(s) this PR fixes**:

Fixes #98285

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/config-common.sh [hunk 1]
```diff
@@ -158,7 +158,7 @@ export WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-k
 # Path for kube-proxy kubeconfig file on Windows nodes.
 export WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
 # Pause container image for Windows container.
-export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.2.1"
+export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.5.1"
```
Comment (pacoxu) [inline cluster/gce/config-common.sh:161]: #98205 would you try to use the image with k8s.gcr.io/pause:3.4.1?

Is gcr.io/gke-release/pause-win:1.5.1 the official one?

Comment (jeremyje) [reply cluster/gce/config-common.sh:161]: Yes, the pause-win image is stored in gke-release.

**Pull request merged**